.markdownlint.yaml

```
@@ -0,0 +1,2 @@
+MD033: false
```

Consider whitelisting which elements to allow:

```suggestion
MD033:
  allowed_elements:
    - figure
    - figcaption
    - img
```

---

Glad to know we can do this! Added.